decoder/ia_core_coder_config.h

```diff
@@ -38,6 +38,23 @@
 #include <impeghd_type_def.h>
 
 #define MAX_ELEMENTS_USAC (50)
+#ifdef LC_LEVEL_4
+// The bit stream extracted value can be 32.
+// Currently this is limited to 3 considering:
+// Audio Channels, Objects and HOA Transport channels.
+#define MAX_NUM_SIGNALGROUPS (28)
+/*The multi‐channel coding tool (MCT) shall not employ
+ * more stereo boxes than 16 for LC profile lvl 3*/
```

Comment:
This comment talks about level 3, we should be writing about level 4

Reply:
Thanks! Will update the comment.